lib/swagger-module.ts

```diff
@@ -131,5 +144,7 @@ export class SwaggerModule {
     );
 
     SwaggerModule.serveStatic(finalPath, app);
+    // covers assets fetched via rel path when swagger url ends with '/'
```

Comment:
Can we add a (reference) link to the original issue? Just so we have more context in the future (on why this is needed)